docs/_scripts/update_preference_docs.py

```diff
@@ -154,7 +154,7 @@ def create_preferences_docs():
         }
 
     text = Template(PREFERENCES_TEMPLATE).render(
-        sections=sections, images_path="../images/_autogenerated/"
+        sections=sections, images_path=DOCS / "images" / "_autogenerated"
```

Comment:
The slashes here are needed because the file structure for the two repos are different when building the docs. For napari/napari, we have `guides/<files>` but for napari/napari.github.io we have `guides/stable/<files>`.

Reply:
Would you add that explanation as a comment in the code here? I think that would be useful to future developers.

Reply:
Let me know if this is clear enough!

Reply:
Seems good to me!